misk-jobqueue-testing/src/main/kotlin/misk/jobqueue/FakeJobHandlerModule.kt

```diff
@@ -0,0 +1,37 @@
+package misk.jobqueue
+
+import misk.inject.KAbstractModule
+import kotlin.reflect.KClass
+
+class FakeJobHandlerModule<T : JobHandler> private constructor(
```

Comment:
i'd rather not have separate modules for testing. seems reasonable that a module with an API also includes the proper fake for testing

Reply:
that is reasonable, though it means you always include testing code. Misk's a bit conflicted in terms of having separated testing modules right now.

Reply:
i think including testing code is the right idea. why wouldn't you want this fake for your tests if you're using a job queue?

Reply:
you'd potentially not want it in a prod artifact. but for testing, definitely.

Reply:
that's a fair point. personally I don't think it's worth it, but we should probably poll the misk audience to get a consensus for consistency